### ISSUE TITLE
[FEAT]: Sub Tabs

### ISSUE THREAD

**Issue by Cleover** [Tormak9970/TabMaster#219]
When hovering over a tab you currently get the option to press the <right-option> button to open up the TabMaster menu for that tab.

It would be really nice to have an option to press the <left-option> button which opens up a similar popup menu to switch to an "alt-filter" for a tab.

Example menu in text when hitting <right-option> hovering over a tab title:
> Default
<Filter 2>
<Filter 3>

(Allowing you to select a different sub tab to show)
Default
<Filter 2>
> <Filter 3>

(Upon selecting a sub tab the tab switches the filter to the selected sub tabs filter)

This could be something which you setup in the current tab config page at the bottom, same filter options etc, just an option to basically make another filter section w/ a sub tab name. (Allowing for tabs which have two or more filers you can easily switch between)

Sub tabs should have names so they can be selected in the menu, but I don't think the name of the tab itself should be changed when switching to a sub-tab, I'm unsure where the current selected tab name could be shown however. (Worst case you have to open the menu to see which is selected perhaps, or the selected sub tab could be added to the end of the current tabs named with brackets around it, although I think it would look visually eh)

I don't like having a lot of tabs which are all duplicates of one another with slight difference in filters as it takes up too much space, but I'd like to have the option to switch between tab configs which only alter things slightly for a tab.

For example, I have a tab called "Achievements" which shows me all games with achievements, and I'd like to have two "Sub Tabs" for "Owned" and "30+"

(Owned meaning games w/achievements not from my family sharing. 30+ just being games with 30 or more achievements)

**Comment by Cleover** [comment]
After thinking on it a bit more it may be nice to instead have tabs be created normally (separately), but shown/hidden tabs can be added as a sub tab to another tab.

Having different sorting for sub tabs would be nice, but at that point it's essentially the full tab config again.

**Comment by jessebofill** [comment]
I don’t think it makes sense to implement something like this because we would have to restructure how tabs work and you can basically already achieve what you are looking for using profiles. Profiles basically just save the configuration of hidden tabs and display order. So you could just create you variant tab as a new tab and create two profiles that switch between which one is hidden. The tab can be the same name if you want. It would also allow for differing app sorting.
